user-guide/Standard_Apps/Monitoring_Solutions/Smart_Trap_Processor/Using_Smart_Trap_Processor/Processor_Configuration.md

```diff
@@ -0,0 +1,167 @@
+---
+uid: Processor_configuration
+---
+
+# Configuring processing rules
+
+To configure a processor element, you can either use DataMiner Cube or the Smart Trap Processor tool (on the *Processor Configuration* page). The configuration described below is the same regardless of which interface is used.
+
+## Trap IP Sources configuration
+
+Before you define any rules for trap processing, you first have to specify the IP address from which traps will be sent that will be received by the DataMiner Agent hosting the processor element.
+
+To do so, set the *Trap IP Sources* parameter to the IP address corresponding with the source of the SNMP traps sent to the DataMiner System.
```

Comment:
Based on what is mentioned below (line 120), it appears that multiple IP addresses can be specified here. Which separator should be used for this?

Reply:
Hi Marieke, I agree that the naming of the processed table should be consistent.
on the Generic Trap Processor Protocol, the name of the table is "Processed Messages".
On the app, the page where the processed data is observed is called "Processed Traps". The data here comes from the same "Processed Messages" table on the protocol. So they essentially mean the same thing.
I would be ok with sticking with "Processed Traps" across the documentation.

For the multiple IP address input, the separator is a comma.